src/cortxfs/cortxfs_ops.c

```diff
@@ -79,19 +79,23 @@ static int __cfs_getattr(struct cfs_fs *cfs_fs, const cfs_cred_t *cred,
 {
```

Comment:
At the end when we will have a cached FH and we will be using FH everywhere, I see cfs_getattr will be removed as it is only getting the stats and that we will be achieving other way:-
`stat = cfs_fh_stat(fh):`